### DISCUSSION TITLE
config help

### DISCUSSION THREAD

**Discussion by mayrsascha** [renovatebot/renovate#11916] [Request Help]
Good afternoon,

I have the following `renovate.json` in the `main` branch in the root folder of my project:

```
{
  "extends": [
    "config:base"
  ],
  "packageRules": [
    {
      "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
      "automerge": true
    }
  ]
}
```

I want minor and patch update Pull Requests to be auto-merged. There I followed the [configuration options outlined in the docs](https://github.com/PascalMayr/rebelz_upsell), but unfortunately, I still have to manually merge minor and patch update PRs generated by Renovatebot.

**Comment by rarkins** [comment]
What do the logs say for the branches you expect to be automerged?

**Comment by mayrsascha** [comment]
@rarkins I assume by logs you mean the comments on the GitHub PR branches, please let me know if there are other logs that I'm not aware of.

Please check these 2 screenshots of PRs that I recently manually merged. FYI I actually merged a couple of more patch update PRs, le me know if you need those as well.

- https://www.collabshot.com/show/c02b14
- https://clb.sh/bb297d

**Reply by rarkins** [reply]
app.renovatebot.com

**Reply by rarkins** [reply]
And make sure you've read https://docs.renovatebot.com/key-concepts/automerge/ until the end

**Comment by mayrsascha** [comment]
@rarkins Thanks, adding `"ignoreTests": true` helped in our case.
